src/components/widgets/TraceTypeSelector.js

```diff
@@ -7,19 +7,22 @@ import {traceTypeToPlotlyInitFigure, renderTraceIcon, plotlyTraceToCustomTrace}
 
 const renderActionItems = (actionItems, item) =>
   actionItems
-    ? actionItems(item).map((action, i) => (
-        <a
-          className="trace-item__actions__item"
-          key={i}
-          aria-label={action.label}
-          data-microtip-position={`top-left`}
-          role="tooltip"
-          href={action.href}
-          target="_blank"
-        >
-          {action.icon}
-        </a>
-      ))
+    ? actionItems(item).map(
+        (action, i) =>
+          !action.onClick ? null : (
```

Comment:
this could just be `action.onClick && x` instead of `!action.onClick ? null : x`. Not a big deal but it jumped out at me for some reason